finding/finding.go

```diff
@@ -96,23 +94,19 @@ const (
 )
 
 // Finding represents a finding.
-//
-//nolint:govet
 type Finding struct {
-	Probe       string             `json:"probe"`
-	Outcome     Outcome            `json:"outcome"`
-	Message     string             `json:"message"`
 	Location    *Location          `json:"location,omitempty"`
 	Remediation *probe.Remediation `json:"remediation,omitempty"`
 	Values      map[string]int     `json:"values,omitempty"`
+	Probe       string             `json:"probe"`
+	Message     string             `json:"message"`
+	Outcome     Outcome            `json:"outcome"`
```

Comment:
So one thing about the non-test structs is these values affect the JSON marshaling order. So these `Finding` and `Location` structs may have been intentionally ordered the way they were. @laurentsimon ?

Reply:
I did not order them in particular order at the time. Do we need to?

Reply:
It's all preference and readability I think. e.g. if we prefer, a slightly more user-friendly order:
```
    {
      "probe": "toolDependabotInstalled",
      "outcome": 12,
      "message": "tool 'Dependabot' is used",
      "location": {
        "type": 1,
        "path": ".github/dependabot.yml",
        "lineStart": 1
      }
    },
```
or
```
    {
      "location": {
        "type": 1,
        "path": ".github/dependabot.yml",
        "lineStart": 1
      }
      "probe": "toolDependabotInstalled",
      "message": "tool 'Dependabot' is used",
      "outcome": 12,
    },
```

Reply:
Systems will use these after someone consumes the data.

Reply:
I'm fine continuing with this change if we want, since this is an unreleased feature.